content/en/platform/corda/5.2/application-networks/upgrading/_index.md

```diff
@@ -11,27 +11,32 @@ menu:
 
 # Upgrading an Application Network
 
-If an administrator [re-registers the MGM]({{< relref "../creating/mgm/reregister.md" >}}) or [upgrades a Corda cluster from 5.0 to 5.1]({{< relref "../../deploying-operating/deployment/upgrading/_index.md" >}}), you should upgrade the member {{< tooltip >}}CPIs{{< /tooltip >}} of your application network. The `MemberInfo` platform version is set at member registration time and so upgrading members ensures that this value reflects the current version. Upgrading members enables the following:
+If an administrator [re-registers the MGM]({{< relref "../creating/mgm/reregister.md" >}}) or [upgrades a Corda cluster from 5.1 to 5.2]({{< relref "../../deploying-operating/deployment/upgrading/_index.md" >}}), you should upgrade the MGM and the member {{< tooltip >}}CPIs{{< /tooltip >}} of your application network. The `MemberInfo` platform version is set at member registration time and so upgrading members ensures that this value reflects the current version. Upgrading members enables the following:
```

Comment:
I am not too sure about the phrasing here. The MGM should re-register on the new platform and members should upgrade their CPIs, for MGM no CPI upgrade is involved. I think I get what is written here, but for clients or users this phrasing could be a bit confusing. I am afraid they will think that the MGM needs to do a CPI upgrade as well.